src/components/card/card.component.ts

```diff
@@ -0,0 +1,113 @@
+import { CustomElement, ICustomElement } from '../../lib/custom-element';
+
+import './card.component.scss';
+
+@CustomElement({
+  selector: 'app-card',
+  // template: `
+  //   <div class="app-card">
+  //     ${name}
+  //   </div>
+  // `,
+  // // Have in account: [:host] style definition only works when useShadow is set to true
+  // style: ``,
+  useShadow: true,
+})
+export class CardComponent extends HTMLElement implements ICustomElement {
+
+  templateElement: HTMLTemplateElement;
+  shRoot: ShadowRoot;
+  useShadow: boolean;
+
+  static get observedAttributes() {
+    return [
+      'character-name',
+    ];
+  }
+
+  constructor() {
+    super();
+    this.useShadow = true;
+    this.templateElement = document.createElement('template');
+    this.shRoot = this.attachShadow({ mode: 'open' });
+  }
+
+  attributeChangedCallback(name: string, oldValue, newValue): void {
```

Comment:
Add types to all params and please add JSDoc to all methods

Reply:
@JuanDiegoLopez Those methods are own of Web Components, therefore I think add JSDoc to them is not strictly necessary, is it?
I'm going to add the types for params 👍 .